classes/video/jsons/future/videoPlaylist.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "Video Playlist",
+  "description": "Class for creating entities containing items 'Video Playlist Item' class."
```

Comment:
```suggestion
  "description": "Class for creating entities containing items from the 'Video Playlist Item' class."
```

Reply:
Same:
Good point, but this is already live...